code/src/components/TodoItem.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import moment from "moment";
+
+import { DeleteButton } from "./DeleteButton";
+import { todos } from "../reducers/todos";
+
+
+export const TodoItem =({ todo }) => {
+    const dispatch = useDispatch()
+
+    return (
+        <Main>
+            <TodoContainer>
+                <div
```

Comment:
Nice solution with the div as the "styled button". never would have thought of that.